src/pynxtools/__init__.py

```diff
@@ -25,6 +25,7 @@
 from pynxtools.definitions.dev_tools.globals.nxdl import get_nxdl_version
 
 LOGGER_LEVELS_TO_HIGHLIGHT = (logging.WARNING, logging.ERROR)
+__version__ = "0.7.2"
```

Comment:
Why do you hard-code the version here?

Reply:
This version is not from VCS. That is defied by developer team, similar to creating a tag before publishing the repo in PyPI. To have a version identical to the git tag, the software allows to check the version in the interpretation stage.

Reply:
What I mean is some mechanism like:
__version__ = importlib.metadata.version("pynxtools")

or so, but not hard-coded in the init file

Reply:
This is what I also don't understand, why do we need to hard-code the version here (and in similar approaches in the other repos)?

Reply:
Yeah this is hard coded.
There are two ways to add the version to a python package such as adding version in `__init__.py` as I did or/and adding the version in the project.metadata using `pyproject.tom` file (as we currently doing). Or both of them can be implemented together. 
if you install `pynxtools` from `PyPI` it will give the same version from `importlib.metadata.version("pynxtools")` as in `__init__.py` file (because `CITATION.cff`, `git tag` and `package version` are check by ci/cd. But, we can not use  
method `importlib.metadata.version("pynxtools")` in __init__.py as that has self dependency.

Reply:
version from `__init__.py` is removed.